src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs

```diff
@@ -135,6 +139,141 @@ void RaiseSecond()
             VisualStudio.Editor.Verify.TextContains("Second?.");
         }
 
+        [WpfFact]
+        [Trait(Traits.Feature, Traits.Features.EditorConfig)]
+        [Trait(Traits.Feature, Traits.Features.CodeActionsFixAllOccurrences)]
+        public void ApplyEditorConfigAndFixAllOccurrences()
+        {
+            var markup = @"
+using System;
+class C
+{
+    public String first;
+    public $$String second;
+}";
+
+            // CodingConventions only sends notifications if a file is open for all directories in the project
+            VisualStudio.SolutionExplorer.OpenFile(new ProjectUtils.Project(ProjectName), @"Properties\AssemblyInfo.cs");
+
+            // Switch back to the main document we'll be editing
+            VisualStudio.SolutionExplorer.OpenFile(new ProjectUtils.Project(ProjectName), "Class1.cs");
+
+            MarkupTestFile.GetSpans(markup, out var text, out ImmutableArray<TextSpan> spans);
+            SetUpEditor(markup);
+            VisualStudio.WaitForApplicationIdle(CancellationToken.None);
+            VisualStudio.Editor.InvokeCodeActionList();
+            VisualStudio.Editor.Verify.CodeActions(
+                new[]
+                {
+                    "Encapsulate field: 'second' (and use property)",
+                    "Preview changes",
+                    "Encapsulate field: 'second' (but still use field)",
+                    "Preview changes",
+                },
+                ensureExpectedItemsAreOrdered: true,
+                verifyCompleteList: true);
+
+            var editorConfig = @"root = true
+
+[*.cs]
+dotnet_style_predefined_type_for_locals_parameters_members = true:warning
```

Comment:
❗️ This needs to be rewritten to test a different diagnostic. This one uses the simplifier analyzer which does not run out-of-process, so the test is currently unable to detect regressions in the OOP handling of .editorconfig.